spec/swagger_helper.rb

```diff
@@ -18,7 +18,8 @@
     "v1/swagger.yaml" => {
       openapi: "3.0.1",
       info: {
-        title: "API V1",
+        title: "LAA Legal Framework",
+        description: "This service provides Legal Framework details to the LAA Apply for legal aid service",
```

Comment:
The data is supposed to be more generally applicable than this, although i do not believe anyone else uses it.

```suggestion
        description: "This service provides read-only lookup data relating to applications for Civil legal aid. [It is primarily used the "Apply for civil legal aid service"]",
```

Reply:
👍  This description based on the repos description, keeping them uniformed